examples/gemma3n/gemma-3n-e2b-vision-audio-qlora.yml

```diff
@@ -0,0 +1,77 @@
+base_model: google/gemma-3n-E2B-it
+processor_type: AutoProcessor
+
+# Automatically upload checkpoint and final model to HF
+# hub_model_id: username/custom_model_name
+
+plugins:
+  - axolotl.integrations.cut_cross_entropy.CutCrossEntropyPlugin
+cut_cross_entropy: true
+
+# for use with fft to only train on language model layers
+# unfrozen_parameters:
+  # - model.language_model.*
+  # - lm_head
+  # - embed_tokens
+
+load_in_4bit: true
+
+# these 3 lines are needed for now to handle vision chat templates w images
+skip_prepare_dataset: true
+remove_unused_columns: false
+sample_packing: false
+
+# gemma3 doesn't seem to play nice with ddp
+ddp_find_unused_parameters: true
+
+chat_template: gemma3n
+eot_tokens:
+  - <end_of_turn>
+
+datasets:
+  - path: Nanobit/text-vision-audio-2k-test  # requires downloading audio/image in advance in README.md
+    type: chat_template
+    data_files:
+      - dataset.jsonl
```

Comment:
I think HF is buggy with uploading audio/image, so this ds config requires downloading the raw files in advance.